include/overlay023/ov23_0224B05C.h

```diff
@@ -61,7 +61,7 @@ void ov23_0224D6AC(int param0, int param1, void * param2, void * param3);
 int ov23_0224D6F4(void);
 BOOL ov23_0224D7C8(int param0);
 BOOL ov23_0224D87C(int param0);
-void ov23_0224D898(int param0);
+void Underground_SecretBaseRemovePlayer(int param0);
```

Comment:
`UndergroundSecretBase_RemovePlayer`?

How would this differ from just `Underground_RemovePlayer`? (I admittedly don't remember a lot of the Underground features)

Reply:
You can dig a secret base into any wall and players can enter/leave it at any time while connected